lib/engine/part/city.rb

```diff
@@ -9,8 +9,8 @@ class City < RevenueCenter
       attr_accessor :reservations
       attr_reader :slots, :tokens
 
-      def initialize(revenue, slots = 1, groups = nil, hide = false, visit_cost = nil)
-        super(revenue, groups, hide, visit_cost)
+      def initialize(revenue, slots = 1, groups = nil, hide = false, visit_cost = nil, optional = false)
```

Comment:
there's too much stuff here now along with the super

can we switch this to dictionary args?

def initialize(revenue, **opts)
then the super call only needs to be super